www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+
+## The Future: O(1) Build Tooling
+
+![webpack vs. snowpack diagram](/img/snowpack-unbundled-example-3.png)
+
+Snowpack is a move away from complex bundled dev environments and towards faster, unbundled development. Unbundled development introduces a new, lighter complexity story for web development where you can move faster by simply doing less unnecessary work.
+
+**Snowpack is a platform for O(1) build tooling.**  This is a term I first heard used by CodeSandbox's [Ives van Hoorne](https://www.youtube.com/watch?v=Yu9zcJJ4Uz0) that I think perfectly captures what makes Snowpack's workflow so great: when a file changes, only that file is rebuilt. There's no larger rebundling or application rebuilding needed, just a simple, linear, deterministic `input -> build -> output` workflow.
+
+- O(1) build tooling is easier to reason about, since your build input is always a single file.
+- O(1) build tooling stays fast in large projects because only changed files are ever rebuilt.
+- Caching is much more effective as well, since file changes never invalidate other cached files.
+
+Pair this with a platform like Snowpack that only builds individual files as requested by the browser, and you get a developer experience that's dramatically lighter-weight than your traditional, bundled dev server.
+
+
+## `dev` - A Better Dev Environment
+
+![dev command output example](/img/snowpack-dev-startup-2.png)
+
+Run `snowpack dev` to start your dev environment, and the first thing you'll notice is **how flippin' fast it is.** Snowpack starts up instantly, usually in less than 20ms. 
+
+That's right, 20ms. 
+
+This incredible speed is possible because Snowpack only builds files during development as they are requested by the browser. On your very first page load, Snowpack will build each requested file and then cache it for future use. Because each file is built individually, that cached file can be reused forever without ever needing a second build (until you change the file yourself, that is).
+
+`snowpack dev` includes a dev server for hosting, a fully customizable build pipeline for building, and a bunch more familiar features right out of the box:
+
+- TypeScript Support
+- JSX Support
+- Hot Module Replacement (HMR)
+- Importing CSS
+- Importing CSS Modules
+- Importing Images & Other Assets
+- Custom Routing
+- Proxying Requests
+
+**Build Scripts** are 1-line integrations to connect your favorite build tools into your Snowpack build. Thanks to this new idea of O(1) build tooling, every file build can be expressed as a deterministic `input -> build -> output` workflow, which makes Snowpack perfect for Linux-style piping through an already-existing CLI. 
```

Comment:
Second sentence is a bit clunky. "O(1) style build tools allow every file build to be expressed as a deterministic `input -> build -> output` workflow. This flow allows for Snowpack to be piped into and out of existing UNIX-y CLI tools"